KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -403,6 +403,8 @@ public enum UITexts {
         static let feesAndTaxesIncluded = "Text.Quote.FeesAndTaxesIncluded".localized
         static let result = "Text.Quotes.Result".localized
         static let results = "Text.Quotes.Results".localized
+        static let errorNoAvailabilityForTheRequestTimeTitle = "Text.Quotes.ErrorNoAvailabilityForTheRequestTimeTitle".localized
```

Comment:
There will be more errors so maybe create for them separate enum inside Qoutes?

Reply:
I'd prefer to keep it this way for now. Maybe we can think about translations reorganization once we'll be preparing the codebase to migrate to common phrase project.

Reply:
I would add the errors in the UITexts.Errors. Maybe have sth like UITexts.Errors.Quote to give them some separation from the other error types?

Reply:
@dianakarh @BartlomiejSopalaKarhoo we should talk about approach, but for now, I'd like to keep it as it is. We may change it later.

Reply:
2 : 1 you have been voted down :)

Reply:
This is such a small issue that I prefer to move on and get this merged into develop asap. Are we aligned with Android on this (similar key names)? We'll be adopting the Android key names when we merge the 2 projects in Phrase anyway, but until then let's at least keep some consistency with the other platform.